pages/advanced-algorithms/available-algorithms/node2vec.mdx

```diff
@@ -118,10 +118,10 @@ The procedure returns calculated embeddings and the corresponding list of embedd
   training progresses.
 - `sg : {0, 1}` ➡ Training algorithm: 1 for skip-gram; otherwise CBOW.
 - `hs : {0, 1}` ➡ If 1, hierarchical softmax will be used for model training. If
-  0, and `negative` is non-zero, negative sampling will be used.
+  0, and `negative` is positive, negative sampling will be used.
```

Comment:
"negative is positive" is very confusing. Please find a more clear way to express the idea.